doc/guides/doc-style-guide.md

```diff
@@ -33,6 +33,11 @@ this guide.
 * Documents must start with a level-one heading.
 * Prefer affixing links (`[a link][]`) to inlining links
   (`[a link](http://example.com)`).
+* Don't use [non-breaking spcaes][] in headings.
```

Comment:
```suggestion
* Don't use [non-breaking spaces][] in headings.
```

Reply:
Ah I fixed this locally but forgot to push. Will push once my compile is done :)